### ISSUE TITLE
Option to modify Ken Burns

### ISSUE THREAD

**Issue by jankete** [mvysny/photocloud-frame-slideshow#74]
Please add the ability to modify the zoom-level and the animation speed of the Ken Burns effect.

**Comment by amingle4** [comment]
Agree!  Love the Ken Burns effect, but at times, it's too fast and too zoomed in.

**Comment by TheMjW** [comment]
I agree as well. For example:
If the screen of the tablet is widescreen and the photo displayed is square, then have the photo zoomed enough to just fill the sides and just pan up and down to display the whole photo. 

If the photo displayed is a panorama making it more wide than the tablet’s screen, then zoom to fill up and down and only pan left and right to show the whole picture. 

That would be quite effective and practical as the photo will not zoom into a wall
